src/pages/validator/amplifier/integrator-onboarding.mdx

```diff
@@ -0,0 +1,87 @@
+# Integrate your chain
+
+## Prerequisites
+
+- Make sure you have `wasm` enabled.
+- You should know how to deploy and interact with a CosmWasm contract. Refer to the [Osmosis CosmWasm testnet deployment tutorial](https://docs.osmosis.zone/cosmwasm/testnet/cosmwasm-deployment/) if you’d like a refresher — just replace `osmosisid` with `axelard` and use the `arm64` version of the [contract optimizer](https://github.com/CosmWasm/optimizer) if you are working on a [Mac with Apple silicon](https://support.apple.com/en-us/116943). Run `--gas auto --gas-adjustment 2` to provide your transactions with sufficient gas.
+
+## Devnet information
+
+- Devnet RPC: `http://devnet.rpc.axelar.dev:26657`
+- Devnet LCD (REST API/JSON RPC): `http://devnet.rpc.axelar.dev:1317`
+- Devnet gRPC: `http://devnet.rpc.axelar.dev:9090`
+- Chain ID: `devnet-wasm`
+- Other info can be found in [`devnet-amplifier.json`](https://github.com/axelarnetwork/axelar-contract-deployments/blob/e6f549de873e0ad211a073218f172d03f6e92c3b/axelar-chains-config/info/devnet-amplifier.json).
+
+## Set up your CosmWasm devnet environment
+
+1. Download the pre-built [`axelard v0.35.5` binary](https://github.com/axelarnetwork/axelar-core/releases/tag/v0.35.5) from GitHub.
+1. Generate an account by running `./bin/axelard keys add test`.
+1. Fund your devnet account with `uwasm` tokens:
+    - Join the [Axelar Discord](https://discord.com/invite/axelar).
+    - Get the Developer Role.
+    - Go to the **#faucet** channel and submit a request with your account to get 100 test tokens. A sample request: `!faucet devnet axelar1suw3fsckht00c5srgtr8p6lhn58e9lxq9yf5th`
+1. Query your account balance by running `axelard q bank balances axelar1suw3fsckht00c5srgtr8p6lhn58e9lxq9yf5th —-node [RPC]`.
+1. Deploy your CosmWasm contract.
+
+## Run `ampd` / `tofnd`
+
+See the [Become an Amplifier Verifier (Worker)](/validator/amplifier/verifier-onboarding) for detailed instructions on how to run ampd and tofnd on your machine.
+
+## Onboard a chain to Amplifier
+
+1. Build all of the deployed contracts, such as the router and service registry, from [`devnet-amplifier.json`](https://github.com/axelarnetwork/axelar-contract-deployments/blob/e6f549de873e0ad211a073218f172d03f6e92c3b/axelar-chains-config/info/devnet-amplifier.json) using the [Cosmwasm `rust-optimizer`](https://github.com/CosmWasm/optimizer). From the root of the repo, run:
```

Comment:
they actually don't need to build the router and the service registry, they only need to build the contracts that they will deploy. In reality, it's easier to just build everything though, but it feels a bit confusing to specifically call out contracts that they actually don't need to build.

Reply:
We can leave a note, but I think what we have here is fine, since it's easier to build everything.

Reply:
This is a separate section though (Setup Amplifier contracts). It's not part of onboarding a new chain. You can put it in another section and add a note that this has already been done and the devnet config points to the deployed address. Note that the building the contracts isn't done from the config (the config stores the addresses after deployment. there's a dedicated [deployment script](https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/cosmwasm/deploy-contract.js) that's used that generates the config.